ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -86,6 +86,8 @@ tests:
     cluster_profile: packet-sno
     test:
     - ref: baremetalds-packet-check
+    env:
+      LEAK_THRESHOLD_SECONDS: 5:0:0
```

Comment:
it's actually more important to apply that for ``assisted-packet-check`` than for ``sno-packet-check``
I'd say we should apply it for both for consistency

Reply:
ah oops, meant to put it on assisted-packet-check

Reply:
sno packet is just for the live-iso stuff, it doesn't need such a long time

Reply:
right